### PR TITLE
Add perfmetrics recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
@jamadden do you agree with being added as maintainer?

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

File: recipes/perfmetrics/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/zodb/perfmetrics
+  summary: Send performance metrics about Python code to Statsd
+  license: BSD-3-Clause
```
Comment (dopplershift) [inline recipes/perfmetrics/meta.yaml:39]: That's not a BSD 3 clause license--it has a 4th clause. Not sure if that's standard or a one-off.

Comment (goanpeca) [reply recipes/perfmetrics/meta.yaml]: Changed to `BSD-3-Clause-Modification`

https://spdx.org/licenses/BSD-3-Clause-Modification.html

Comment (dopplershift) [reply recipes/perfmetrics/meta.yaml]: :+1:

**Review by dopplershift — APPROVED** [review]

**Pull request merged**